### DISCUSSION TITLE
Excluding all info other than SRR ID

### DISCUSSION THREAD

**Discussion by GitTodojetaime** [sandbox-bio/sandbox.bio#107] [Q&A] [answered]
I am on lesson 1 exercise and struggling to exclude all information in my output that is not the SRR ID. Here are my commands:

touch read.names.txt (creating the file for NA12878.fastq sequence data)
seqkit seq  --only-id NA12878.fastq | head (extracting the sequence SRR ID only)

I am very new to this. Thank you for any suggestions!

**Comment by robertaboukhalil** [comment, marked as answer]
Hello! A few pointers:

* What does `seqkit seq --only-id NA12878.fastq | head` output? Does it output only the SRR IDs, or also the rest of the FASTQ entry? 
* Refer to https://sandbox.bio/tutorials/seqkit-intro/3 for info on how to extract just the IDs and not the rest of the FASTQ entry

Hope this helps!

**Reply by GitTodojetaime** [reply]
It outputs all of the rest of the FASTQ entry.

**Reply by GitTodojetaime** [reply]
So I tried this.. seqkit seq --name --only-id hairpins.fa | head..after opening the file the sequencing read names were saved in. I generated an output of ID's only that looks correct but my tutorial isn't marking it as correct. Let me know what you think.

**Reply by robertaboukhalil** [reply]
Ah, looks like you're using the wrong file! It should be `NA12878.fastq`, not `hairpins.fa`, so `seqkit seq --only-id --name NA12878.fastq > read_names.txt`
